extensions/0832/SuperAlert.js

```diff
@@ -0,0 +1,317 @@
+                        arguments: {
+                            T: {
+                                type: Scratch.ArgumentType.STRING,
+                                menu: 'textOptions',
+                                defaultValue: 'Title',
+                            },
+                            C: {
+                                type: Scratch.ArgumentType.COLOR,
+                                defaultValue: '#FFFFFF',
+                            },
+                        },
+                    },
+                    {
+                        opcode: 'setTextSize',
+                        blockType: Scratch.BlockType.COMMAND,
+                        text: Scratch.translate({ id: 'set_text_size', default: 'Set [T] text size to [S]px' }),
+                        arguments: {
+                            T: {
+                                type: Scratch.ArgumentType.STRING,
+                                menu: 'textOptions',
+                                defaultValue: 'Title',
+                            },
+                            S: {
+                                type: Scratch.ArgumentType.NUMBER,
+                                defaultValue: 24,
+                            },
+                        },
+                    },
+                    {
+                        opcode: 'alignTitleText',
+                        blockType: Scratch.BlockType.COMMAND,
+                        text: Scratch.translate({ id: 'align_title', default: 'Align title text to [A]' }),
+                        arguments: {
+                            A: {
+                                type: Scratch.ArgumentType.STRING,
+                                menu: 'alignmentOptions',
+                                defaultValue: 'center',
+                            },
+                        },
+                    },
+                    {
+                        opcode: 'alignContentText',
+                        blockType: Scratch.BlockType.COMMAND,
+                        text: Scratch.translate({ id: 'align_content', default: 'Align content text to [A]' }),
+                        arguments: {
+                            A: {
+                                type: Scratch.ArgumentType.STRING,
+                                menu: 'alignmentOptions',
+                                defaultValue: 'center',
+                            },
+                        },
+                    },
+                ],
+                menus: {
+                    textOptions: [
+                        { text: 'Title', value: 'Title' },
+                        { text: 'Content', value: 'Content' },
+                    ],
+                    alignmentOptions: [
+                        { text: 'Left', value: 'left' },
+                        { text: 'Center', value: 'center' },
+                        { text: 'Right', value: 'right' },
+                    ],
+                    fontOptions: [
```

Comment:
Be careful with font options, as not all built-in fonts are supported on every device. Preferably, the extension could run off of fonts from the VM, but I'm not sure if that's possible, so the current solution is alright.